src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/hotel?useSSL=false&characterEncoding=UTF-8&zeroDateTimeBehavior=convertToNull&serverTimezone=UTC
```

Comment:
Add file to .gitignore, please.

Reply:
It is in my project already. But seems not to be working.

Reply:
The .gitignore file is the one that's generated. We can add the following ```./src/main/resources/application.properties``` to ignore that specific file